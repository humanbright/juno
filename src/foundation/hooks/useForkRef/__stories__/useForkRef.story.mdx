<!-- This file generate automatically, should not modify that directly -->

import { Canvas, Meta, Story } from '@storybook/addon-docs';

<Meta title="API Reference/Hooks/useForkRef" />

## useForkRef

### Variables

- [useForkRef](#const-useforkref)

## Variables

### `Const` useForkRef

â€¢ **useForkRef**: _useForkRef_ = MuiUseForkRef

That method will give you ability fork same value `ref object` or `method`
Example:

```ts
const refA = useRef();
const refB = useRef();

const ref = useForkRef(refA, refB);

useEffect(() => {
  console.log(refA); // get that current div dom
  console.log(refB); // also get that current div dom
}, []);

return <div ref={ref} />;
```
