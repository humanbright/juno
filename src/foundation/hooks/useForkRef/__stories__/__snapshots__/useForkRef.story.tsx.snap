// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots ðŸ”§ Foundation/Hooks/useForkRef useForkRef 1`] = `
.c0 {
  font-size: 1.5rem;
  font-weight: 400;
  font-family: Lato,Helvetica,Arial,sans-serif;
  line-height: 32px;
  color: #121212;
}

<div>
  <h5
    className="MuiTypography-root c0 MuiTypography-paragraph MuiTypography-displayBlock"
    data-color="neutral.f06"
    data-variant="headline1"
  >
    view console to get ref value
  </h5>
</div>
`;
